src/net/connection.rs

```diff
@@ -1801,6 +1801,7 @@ mod test {
         pinger.join().unwrap();
     }
 
+    #[ignore] // fails intermittently when run via `cargo test`
```

Comment:
Can you open a separate issue for this?

Reply:
Sure -- should I remove the `#[ignore]` in this PR?

Reply:
No, you can leave the `#[ignore]`.  I just want to make sure we have an issue open to track this bug.